app/controllers/site_controller.rb

```diff
@@ -1,5 +1,10 @@
 # SiteController handles the frontend for the website
 class SiteController < ApplicationController
   def index
+    # show the front page
+  end
+
+  def redirect
```

Comment:
I'd suggest the sitecontroller is used for the 'sessionless' site only, with only static content.